doc/source/whatsnew/v1.1.0.rst

```diff
@@ -529,6 +529,7 @@ I/O
 - Bug in :meth:`DataFrame.to_sql` where an ``AttributeError`` was raised when saving an out of bounds date (:issue:`26761`)
 - Bug in :meth:`read_excel` did not correctly handle multiple embedded spaces in OpenDocument text cells. (:issue:`32207`)
 - Bug in :meth:`read_json` was raising ``TypeError`` when reading a list of booleans into a Series. (:issue:`31464`)
+- Bug in :meth:`to_json` was raising `AttributeError` with column or Series of type `Period` (:issue:`31917`).
```

Comment:
"type `Period`" -> "``PeriodDtype``", no period at the end, double-ticks around AttributeError